packages/plugin-php/index.js

```diff
@@ -0,0 +1,26 @@
+import { PHP } from '@octolinker/helper-grammar-regex-collection';
+import liveResolverQuery from '@octolinker/resolver-live-query';
+
+export default {
+  name: 'PHP',
+
+  resolve(path, [target]) {
+    if (target.includes('\\')) {
+      return;
```

Comment:
I'm not a PHP developer, so I don't know if there is a case where core class contains a slash as well ... I'll investigate

Reply:
There are no namespaced classes in the php std lib